internal/jimm/watcher.go

```diff
@@ -343,9 +343,9 @@ func (w *Watcher) watchController(ctx context.Context, ctl *dbmodel.Controller)
 						machines++
 						cores += n
 					}
-					m.Cores = cores
-					m.Machines = machines
-					m.Units = int64(len(v.units))
+					// m.Cores = cores
+					// m.Machines = machines
+					// m.Units = int64(len(v.units))
```

Comment:
This is left commented out?

Reply:
this was then removed all togethere in the commit after, so no worries